packages/firestore/src/local/simple_db.ts

```diff
@@ -78,7 +78,7 @@ export class SimpleDb {
       // suggests IE9 and older WebKit browsers handle upgrade
       // differently. They expect setVersion, as described here:
       // https://developer.mozilla.org/en-US/docs/Web/API/IDBVersionChangeRequest/setVersion
-      const request = window.indexedDB.open(name, version);
+      const request = indexedDB.open(name, version);
```

Comment:
I am surprised this works with the IndexedDB shim. We only register on `window.indexedDB`:

https://github.com/firebase/firebase-js-sdk/blob/631a0ec5982b36382a96c15be9916bf29f98a01f/packages/firestore/test/util/test_platform.ts#L57

The CI is green though.